hasura.planx.uk/migrations/1747989699392_run_sql_migration/up.sql

```diff
@@ -0,0 +1,42 @@
+DROP VIEW "public"."teams_summary";
+
+CREATE OR REPLACE VIEW "public"."teams_summary" AS 
+ SELECT t.id,
+    t.name,
+    t.slug,
+    ts.reference_code,
+    ts.homepage,
+    t.domain AS subdomain,
+    ti.has_planning_data AS planning_data_enabled,
+    ts.has_article4_schema AS article_4s_enabled,
+    jsonb_build_object('helpEmail', ts.help_email, 'helpPhone', ts.help_phone, 'emailReplyToId', ts.email_reply_to_id, 'helpOpeningHours', ts.help_opening_hours) AS govnotify_personalisation,
+    (ti.staging_govpay_secret IS NOT NULL) AS govpay_enabled_staging,
+    (ti.production_govpay_secret IS NOT NULL) AS govpay_enabled_production,
+    ts.submission_email AS send_to_email_address,
+    ti.staging_bops_submission_url AS bops_submission_url_staging,
+    ti.production_bops_submission_url AS bops_submission_url_production,
+    tt.logo,
+    tt.favicon,
+    tt.primary_colour,
+    tt.link_colour,
+    tt.action_colour,
+    (
+        (ti.staging_file_api_key IS NOT NULL) AND
+        (ti.staging_power_automate_api_key IS NOT NULL) AND
+        (ti.power_automate_webhook_url IS NOT NULL)
+    )::BOOLEAN AS power_automate_enabled_staging,
+    (
+        (ti.production_file_api_key IS NOT NULL) AND
+        (ti.production_power_automate_api_key IS NOT NULL) AND
+        (ti.power_automate_webhook_url IS NOT NULL)
+    )::BOOLEAN AS power_automate_enabled_production,
```

Comment:
These are now columns per-env, with a conditional query in the frontend which means we can correctly display the values for a given environment.